proto/cline/models.proto

```diff
@@ -129,9 +129,9 @@ enum ApiProvider {
   TOGETHER = 10;
   DEEPSEEK = 11;
   QWEN = 12;
-  DOUBAO = 13;
-  MISTRAL = 14;
-  VSCODE_LM = 15;
+  QWEN_CODE = 13;
```

Comment:
Can we please add QWEN_CODE to the end of the list? Keeping this list in alphabetical order is not required